hosted-ce-setup.sh

```diff
@@ -13,18 +15,39 @@ if [[ $? -eq 0 ]]; then
   cp /tmp/99-local.ini /etc/osg/config.d/99-local.ini
 fi
 
-# need to programmatically get users
-for user in $(echo $CE_USERS | tr ',' ' '); do
-  echo "Adding user $user"
-  adduser $user 
-  mkdir -p /home/$user/.ssh
-  chown $user: /home/$user/.ssh
-  ssh-keyscan -H $(echo $ENDPOINT | cut -d'@' -f2) >> /home/$user/.ssh/known_hosts
-done
-
-echo "Keyscanning.."
-mkdir -p ~/.ssh
-ssh-keyscan -H $(echo $ENDPOINT | cut -d'@' -f2) >> ~/.ssh/known_hosts
+setup_ssh_config () {
+  echo "Adding user ${ruser}"
+  ssh_dir="~${ruser}/.ssh"
+  if [[ $(getent passwd "${ruser}" -ne 0) ]]; then
+      # setup user and SSH dir
+     adduser "${ruser}"
+     mkdir -p $ssh_dir
```

Comment:
This doesn't work: $ssh_dir won't be reexpanded, so it will try to make a directory literally called "~username".